apps/builder/lostpixel.config.ts

```diff
@@ -0,0 +1,9 @@
+import * as process from "process";
+
+export const config = {
```

Comment:
the idea to use typescript here is to actually typecheck the config, but we are not importing lost-pixel types here to use them, I think we should use the types from lost pixel like in the docs https://docs.lost-pixel.com/user-docs/setup/project-configuration#typescript

or just use json/js config, because what's the point of using ts here otherwise?

Reply:
I think we had lost pixel in deps before and it was a big one, so kinda only makes sense if we can install lost pixel types separately from the library, otherwise I would just use json for now 

cc @d-ivashchuk

Reply:
For example `apiKey` does not exist in any of the types excluded in `lost-pixel`. I will keep it as a `.js` file for now.